src/Smalot/PdfParser/Pages.php

```diff
@@ -36,14 +36,12 @@
 class Pages extends PDFObject
 {
     /**
-     * @param bool $deep
-     *
      * @todo Objects other than Pages or Page might need to be treated specifically in order to get Page objects out of them,
      * see https://github.com/smalot/pdfparser/issues/331
      *
-     * @return array
+     * @return array|string
```

Comment:
Why did you add `string` here?

Reply:
I misunderstood what object is being fetched with the `->get` call. Added PHPDoc to make it more clear and refactored the return type.